Assets/Scripts/Algorithms/Planning/LaunchAnglePlanner.cs

```diff
@@ -22,9 +22,13 @@ public struct LaunchAngleOutput {
   // Time to reach the target position in seconds.
   public float TimeToPosition { get; }
 
-  public LaunchAngleOutput(float launchAngle, float timeToPosition) {
+  // Distance from origin to target in meters.
+  public float DistanceToTarget { get; }
```

Comment:
Why is this needed?

Reply:
Uhh leftover from earlier implementation. Good catch, thanks.